src/main/java/org/jabref/gui/util/DefaultFileUpdateMonitor.java

```diff
@@ -87,6 +87,7 @@ private void notifyAboutChange(Path path) {
     public void addListenerForFile(Path file, FileUpdateListener listener) throws IOException {
         if (isActive()) {
             // We can't watch files directly, so monitor their parent directory for updates
+            // toAbsolutePath() will add the path to Jabref as home directory to file, if file is not a absolute path
```

Comment:
I don't think this comment should be here.
If you give this method a relative path it is implementation-dependent, I don't think it is always going to be JabRef's home directory. I guess that in practice we should avoid giving it an absolute path.